org.aegisub.Aegisub.yml

```
@@ -0,0 +1,158 @@
+#   https://github.com/scx/aegisub-flatpak
+
+id: org.aegisub.Aegisub
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: aegisub
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=home
+rename-desktop-file: aegisub.desktop
+rename-appdata-file: aegisub.appdata.xml
+rename-icon: aegisub
+# TODO: Consider separate locales
+separate-locales: false
+cleanup:
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/share/pkgconfig"
+  - "/share/aclocal"
+  - "/man"
+  - "/share/man"
+  - "/share/doc"
+  - "/share/gtk-doc"
+  - "*.la"
+  - "*.a"
+modules:
+  # TODO: Use new build system after it's released.
+  #   buildsystem: meson
+  #   config-opts:
+  #     # Build
+  #     - --buildtype=release
+  #     # - --strip=true
+  #     - -Db_lto=true
+  #     - -Ddefault_library=static
+  #     # Audio output
+  #     - -Dalsa=enabled
+  #     - -Dlibpulse=enabled
+  #     - -Ddefault_audio_output=ALSA
+  #     # Video source
+  #     - -Dffms2=enabled
+  #     - -Davisynth=enabled
+  #     # Text
+  #     - -Dhunspell=enabled
+  #     - -Duchardet=enabled
+  #     - -Dcsri=enabled
+  #     # Other
+  #     - -Dfftw3=enabled
+  #     - -Dsystem_luajit=true
+  #     - -Dlocal_boost=false
+  #     - -Dwx_version=3.2.4
+  #     - -Dcredit="zefr0x"
+  - name: aegisub
+    buildsystem: autotools
+    build-options:
+      strip: true
+    config-opts:
+      - --prefix=/app
+      - --enable-lto
+      - --with-boost-libdir=/app/lib
+      # FIX: Use system luajit (can't be detected with the used build options).
+      # - --with-system-luajit=yes
+      - --with-build-credit=zefr0x
+      - --disable-update-checker
+    sources:
+      - type: git
+        url: https://github.com/wangqr/Aegisub
+        tag: v3.3.3
+        commit: a596a88115890a6f8ecd41518f1dbcdea1f6cd61
+        x-checker-data:
+          type: git
+          tag-pattern: ^v(\d.\d.\d)$
+          version-scheme: semantic
+      - type: patch
+        path: fix-and-improve-appstream-file.patch
+      - type: patch
+        # This commit is needed to enable wayland support.
+        # It was disabled since wxwidgets wasn't supporting wayland in older releases.
+        # After the v3.3.3 release it was enabled but not released since the developer is not active.
+        # Without it the app might crash on XWayland.
+        path: enable_wayland.patch
+      - type: patch
+        # Needed to link with latest version of libboost.
+        # Source: https://gitlab.archlinux.org/archlinux/packaging/packages/aegisub/-/blob/fef4a8f94ce2fc2e450b608f926a5fd44215e653/boost-1.81.0.patch
+        path: boost-1.81.0.patch
+    modules:
+      - wxwidgets.yaml
+      - ffms2.yaml
+      - boost.yaml
+      # TODO: Install luajit with all needed libraries in the system rathen then using bundled.
+      # - luajit.yaml
+      - ./shared-modules/linux-audio/fftw3f-static.json
```

```suggestion
      - ./shared-modules/linux-audio/fftw3f.json
```

don't use the static version. it's reserved for audio plugins that can't dynamic link.